src/libs/Navigation/AppNavigator/BaseDrawerNavigator.js

```diff
@@ -53,7 +53,7 @@ class BaseDrawerNavigator extends Component {
         const content = (
             <Drawer.Navigator
                 backBehavior="none"
-                key={`BaseDrawerNavigator${this.props.isSmallScreenWidth}`}
+                key="BaseDrawerNavigator"
```

Comment:
Why does this need a key ? 🤔

Reply:
lol good point

Reply:
I mean I assume there is some reason. but Not sure why and less sure why it was changed...

I think the idea is that the key would cause react to remount the drawer navigator if the `isSmallScreenWidth` prop changes (unclear why), but the key is hardcoded now so I think it's pointless?

Reply:
It was introduced [here](https://github.com/Expensify/App/pull/6356) to force the navigator to unmount and fit orientation changes in mobile devices. We then removed it since that bug was apparently fixed in a later version of `react-navigation` as described [here](https://github.com/Expensify/App/issues/8840#issuecomment-1133425957). This key is now useless and we should have just removed it altogether haha